black_it/utils/json_pandas_checkpointing.py

```diff
@@ -166,6 +173,10 @@ def save_calibrator_state(  # pylint: disable=too-many-arguments,too-many-locals
     with open(checkpoint_path / "calibration_params.json", "w") as f:
         json.dump(calibration_params, f, cls=NumpyArrayEncoder)
 
+    # save random generator state in a json dictionary
+    with open(checkpoint_path / "random_generator_state.json", "w") as f:
+        json.dump(random_generator_state, f)
```

Comment:
Is this really necessary? after all we are saving the same "random_generator_state" also inside the "calibration_params" dictionary (line 166)

Reply:
@marcofavoritobi